magento/graphql-ce
# GraphQL - Added sort by options to Products GraphQL type

### Description
Added available sort options and default sort option for product listing to page_info object of Products GraphQL object.

### Fixed Issues (if relevant)
1. magento/graphql-ce#3: Return 'Available sort options' for product listing

### Manual testing scenarios
1. Add following object to query object page_info and run:
`sort_fields {
        default
        options {
          key
          label
        }
      }`
2. Magento should returns object sort_fields contains default and options 

### Contribution checklist
